src/features/pathway/api.js

```
@@ -0,0 +1,122 @@
+) => {
+  const { data } = await apiClient.post(
+    `/api/pathways/${encodePathwayName(pathwayName)}/years/${year}/building-events`,
+    {
+      new_buildings: newBuildings,
+      demolished_buildings: demolishedBuildings,
+    },
+  );
+  return data;
+};
+
+export const applyTemplatesToYear = async (pathwayName, year, templateNames) => {
```

ESLint

[error] Replace `pathwayName,·year,·templateNames` with `⏎··pathwayName,⏎··year,⏎··templateNames,⏎`

(prettier/prettier)